ADDRESSING.md

```diff
@@ -20,16 +18,21 @@
 
 ## TL;DR
 
-Native URI can be used for addressing:
-- immutable content:  `ipfs://{cid}`
-- mutable content behind IPNS record signed with a specific libp2p key: `ipns://{libp2p-key}` 
-- mutable content behind human-readable DNSLink (DNS TXT record): `ipns://{fqdn}` 
+For immutable content use an IPFS URI (`ipfs://{cid}`).
+- [ipfs://bafkreigh2akiscaildcqabsyg3dfr6chu3fgpregiymsck7e7aqa4s52zy](ipfs://bafkreigh2akiscaildcqabsyg3dfr6chu3fgpregiymsck7e7aqa4s52zy)
+
+For mutable content use an IPNS DNSLink URI (`ipns://{fqdn}`).
+- [ipns://webui.ipfs.io](ipns://webui.ipfs.io)
+
+For web browsers and other tools that do not yet recognise the `ipfs://` and `ipns://` schemes, additionally provide an HTTP-to-IPFS gateway URL.
+- [https://dweb.link/ipfs/bafkreigh2akiscaildcqabsyg3dfr6chu3fgpregiymsck7e7aqa4s52zy](https://dweb.link/ipfs/bafkreigh2akiscaildcqabsyg3dfr6chu3fgpregiymsck7e7aqa4s52zy)
+- [https://dweb.link/ipns/webui.ipfs.io](https://dweb.link/ipns/webui.ipfs.io)
```

Comment:
Webui requires CORS so would show "error" by default for new users, this may be a better example for demo purposes:
```suggestion
- [https://dweb.link/ipns/cid.ipfs.io](https://dweb.link/ipns/cid.ipfs.io)
```